spring-webclient-mocking/src/test/java/com/baeldung/webclientmocking/service/EmployeeServiceTest.java

```diff
@@ -0,0 +1,120 @@
+package com.baeldung.webclientmocking.service;
+
+import com.baeldung.webclientmocking.domain.Employee;
+import com.baeldung.webclientmocking.enums.Role;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.exceptions.base.MockitoException;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+@RunWith(JUnitPlatform.class)
+public class EmployeeServiceTest {
+
+    EmployeeService employeeService;
```

Comment:
This should be `private`?

Reply:
Can you also use `@InjectMocks` on this to avoid having to explicitly initialize it in the `@BeforeEach`?